pkgs/development/libraries/pipewire/default.nix

```diff
@@ -100,6 +101,15 @@ let
       ./0090-pipewire-config-template-paths.patch
       # Place SPA data files in lib output to avoid dependency cycles
       ./0095-spa-data-dir.patch
+      # Fix attempt to put system service units into pkgs.systemd.
+      (fetchurl {
```

Comment:
You need to use `fetchpatch` to clean up the patch since Git forges sometimes change the metadata. Keep in mind that the hash will need to be changed (though you might not be told by Nix due to the FOD hash now existing in your store).

Reply:
Done.